src/requestManager.cpp

```diff
@@ -131,8 +131,8 @@ void CRequestManager::AskFor(const CInv& obj, CNode* from, int priority)
       // if (result.second)  // means this was inserted rather than already existed
       // { } nothing to do
       data.priority = max(priority,data.priority);
-      data.AddSource(from);
-      LogPrint("blk", "%s available at %s\n", obj.ToString().c_str(), from->addrName.c_str());
+      if (data.AddSource(from))
+        LogPrint("blk", "%s available at %s\n", obj.ToString().c_str(), from->addrName.c_str());
```

Comment:
IDK that we did.  I thought it was part of Allman style, but I guess its not.  Allman just covers placement of parens.  Personally I don't mind them.  But we should argue this out and update developer-notes.md.  Anyway I will fix here.